src/main/resources/lang/default.lang

```diff
@@ -1980,6 +1980,18 @@ transform reasons:
 	unknown: unknown
 	infection: infection, villager infection
 
+# -- Teleport Flags --
+teleport flags:
+	retain_open_inventory: retain open inventory, opened inventory, open inventory, inventory
+	retain_passengers: retain passengers, passengers, passenger
+	retain_vehicle: retain vehicle, vehicle
+	retain_direction: retain direction, direction
+	retain_pitch: retain pitch, pitch
+	retain_yaw: retain yaw, yaw
+	retain_x: retain x, x
+	retain_y: retain y, y
+	retain_z: retain z, z
```

Comment:
I think this should be `retain x-coord`

Reply:
See https://github.com/SkriptLang/Skript/pull/6498#discussion_r1527218604